### PR TITLE
make rspec use the format documentation when running any suite

### PR DESCRIPTION
**Pull request by ph**
We suite from the default dot formatter to the documentation format,
this will help localize some error in the build

### CODE REVIEW

**Comment by colinsurprenant** [comment]
why do we pass parameters using `.rspec` instead of using env vars?

**Comment by ph** [comment]
@colin good question, I will move it to env. We were already using the .rspec file in the `travis.yaml` I didn't ask myself why we were doing it.

File: .gitignore [hunk 1]
```diff
@@ -42,3 +42,4 @@ Gemfile
 logs
 qa/integration/services/installed/
 .gradle
+.rspec
```
Comment (colinsurprenant) [inline .gitignore]: is this still needed?

**Comment by colinsurprenant** [comment]
minor comment, did not test, LGTM.

**Comment by ph** [comment]
Weird Its doesn't seem to produce any output on travis, will check again. It should be the right variable.

**Comment by colinsurprenant** [comment]
weird. `SPEC_OPTS` is the right env var and `export SPEC_OPTS="--order rand --format documentation"` should work - it works locally for me. 

Maybe it has to do with the way Travis executes the yml steps?

I see Travis provides support for setting env var https://docs.travis-ci.com/user/environment-variables/ - maybe use that?

**Comment by ph** [comment]
Yeah I will try that, I just need to add them to the existing env. https://github.com/elastic/logstash/blob/master/.travis.yml#L11-L15

**Comment by ph** [comment]
@colinsurprenant Travis' environment variable works like a charm.

**Comment by colinsurprenant** [comment]
@ph ah! perfect!

**Comment by colinsurprenant** [comment]
LGTM :shipit:

**Pull request closed without merging**